components/Sections/MainHeader.tsx

```
@@ -5,33 +5,107 @@ import { Button } from '@/components/ui/button';
+ * - Customizable through HERO_CONFIG constant
+ */
+const MainHeader: React.FC<MainHeaderProps> = () => {
     return (
-        <section className="relative h-[60vh] md:h-[70vh] w-full flex items-center justify-center text-center text-white overflow-hidden">
-            <div className="absolute inset-0 z-0"> 
+        <header 
+          className={styles.hero}
+          role="banner"
+          aria-label="Main hero section"
+        >
+            {/* Background image with overlay */}
```

Biome (2.1.2)

[error] Interactive elements should not be assigned non-interactive roles.

WAI-ARIA roles should not be used to convert an interactive element to a non-interactive element.
Wrap your interactive element in a <div> with the desired role or put the content inside your interactive element.
Unsafe fix: Remove the role attribute.


(lint/a11y/noInteractiveElementToNoninteractiveRole)